packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -444,11 +450,21 @@ contract Deploy is Deployer {
                 _implementation: disputeGameFactory,
                 _data: abi.encodeCall(
                     DisputeGameFactory.initialize,
-                    (cfg.finalSystemOwner())
+                    (tx.origin)
```

Comment:
We should be able to use `msg.sender` - other deployment functions use it. Perhaps we want to have a global `internal sender` or something like that and use that variable everywhere since a design goal is to always use the same deployer for simplicity, we dont need to make that change in this PR